integrations/BingAds/browser.js

```diff
@@ -56,6 +56,13 @@ class BingAds {
     if (value) {
       event.gv = value;
     }
+    else if(revenue) {
```

Comment:
let's use if's only to keep parity with others.
Also update the doc accordingly, the highest preference is total > revenue > value

Reply:
Please check it once more. If any other changes is needed or not.